packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -280,19 +280,19 @@ class HorizontalOverflow extends Foundation<
      */
     private getPositionData = (): PositionChange => {
         const scrollPosition: number = this.getScrollPosition();
-        const distanceRemaining: number =
-            this.horizontalOverflowItemsRef.current.scrollWidth - scrollPosition;
 
-        const scrollPositionIsAtBeginning: boolean = scrollPosition === 0;
-        const scrollPositionIsAtEnd: boolean =
-            distanceRemaining === this.horizontalOverflowItemsRef.current.clientWidth;
-        const scrollPositionIsInMiddle: boolean =
-            !scrollPositionIsAtEnd && !scrollPositionIsAtBeginning;
+        const isAtBeginning: boolean = scrollPosition === 0;
+        const isAtEnd: boolean =
+            this.horizontalOverflowItemsRef.current.scrollWidth - scrollPosition ===
+            this.horizontalOverflowItemsRef.current.clientWidth;
 
-        return {
-            start: scrollPositionIsInMiddle ? false : scrollPositionIsAtBeginning,
-            end: scrollPositionIsInMiddle ? false : scrollPositionIsAtEnd,
```

Comment:
looks like you just need to change the false here to `true`

Reply:
I kinda prefer the changed version, a bit more compact

Reply:
Actually true, because when you are in the middle and there is overflow you can scroll in either direction.

Reply:
Most common use of this is to control visibility of scroll buttons I'd expect, and when you are in the middle you want both because there is content on either side.

Reply:
The documentation for start and end interface is currently as follows:
```
/**
 * Scroll interface for consumers
 * 'start' is when the horizontal overflow scroll is all the way left in LTR (all the way right in RTL)
 * 'end' is when the horizontal overflow scroll is all the right in LTR (all the way left in RTL)
 * TODO #1177: Change interface property names to be more meaningful
 */
```

I would expect then if you are at the beginning start would be `true`, and vice versa for end.

Reply:
In fact, I'm not sure why this couldn't simple be:
```
return {
    start: isAtStart,
    end: isAtEnd
}
```

Reply:
But if you read down this is what we say about overflow start/end.  
 * 'overflowStart' is true when there are items to the left in LTR (right in RTL)
 * 'overflowEnd' is true when there are items to the right in LTR (left in RTL)

So we have two interfaces that report essentially the same information but the meanings of true/false flipped?

Reply:
I updated the PR to reflect but I really think we need to revisit this interface on the next version bump, I don't see the value of having both callbacks.

Reply:
There is already an issue for deprecating scroll position - https://github.com/microsoft/fast-dna/issues/1178